colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+        R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+"""
+
+import numpy as np
+
+from colour.utilities import tsplit, tstack, CaseInsensitiveMapping
+from colour.models.rgb import REC_2020_COLOURSPACE
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2016 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Development'
+
+__all__ = ['WEIGHT',
+           'RANGE',
+           'rgb_to_YCbCr',
+           'YCbCr_to_rgb',
+           'rgb_to_YcCbcCrc',
+           'YcCbcCrc_to_rgb']
+
+WEIGHT = (CaseInsensitiveMapping(
+        {'Rec.601': [0.299, 0.114],
+         'Rec.709': [0.2126, 0.0722],
+         'Rec.2020': [0.2627, 0.0593]}))
+"""
+List of preset luma weightings:
+    'Rec.601' : [0.299, 0.114]
+    'Rec.709' : [0.2126, 0.0722]
+    'Rec.2020' : [0.2627, 0.0593]
+"""
+
+RANGE = (CaseInsensitiveMapping(
+        {'legal_12': [256./4095, 3760./4095],
+         'legal_12_YC': [256./4095, 3760./4095, 256./4095, 3840./4095],
+         'legal_12_int': [256, 3760],
+         'legal_12_YC_int': [256, 3760, 256, 3840],
+         'legal_10': [64./1023, 940./1023],
+         'legal_10_YC': [64./1023, 940./1023, 64./1023, 960./1023],
+         'legal_10_int': [64, 940],
+         'legal_10_YC_int': [64, 940, 64, 960],
+         'legal_8': [16./255, 235./255],
+         'legal_8_YC': [16./255, 235./255, 16./255, 240./255],
+         'legal_8_int': [16, 235],
+         'legal_8_YC_int': [16, 235, 16, 240],
+         'full': [0.0, 1.0],
+         'YPbPr': [0.0, 1.0, -0.5, 0.5]}))
+"""
+List of preset ranges:
+    'legal_12' : [256./4095, 3760./4095]
+    'legal_12_YC' : [256./4095, 3760./4095, 256./4095, 3840./4095]
+    'legal_12_int' : [256, 3760]
+    'legal_12_YC_int' : [256, 3760, 256, 3840]
+    'legal_10' : [64./1023, 940./1023]
+    'legal_10_YC' : [64./1023, 940./1023, 64./1023, 960./1023]
+    'legal_10_int' : [64, 940]
+    'legal_10_YC_int' : [64, 940, 64, 960]
+    'legal_8' : [16./255, 235./255]
+    'legal_8_YC' : [16./255, 235./255, 16./255, 240./255]
+    'legal_8_int' : [16, 235]
+    'legal_8_YC_int' : [16, 235, 16, 240]
+    'full' : [0.0, 1.0]
+    'YPbPr' : [0.0, 1.0, -0.5, 0.5]
+"""
+
+
+def rgb_to_YCbCr(rgb, K=WEIGHT['Rec.709'], inRange=RANGE['full'],
+                 outRange=RANGE['legal_10_YC']):
+    """
+    Converts an array of R'G'B' values to the corresponding Y'CbCr values.
+
+    Note: For ITU-R BT.2020 (Rec.2020) the rgb_to_YCbCr function is only
+    applicable tothe non-constant luminance implementation.
+    The rgb_to_YcCbcCrc function should be used for the constant luminance case
+    See https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+    R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+
+    Parameters
+    ----------
+    rgb : array_like
+        Input R'G'B' values. These may be floats or integers.
+    K : array of two values
```

Comment:
`K : array of two values`: should be `K : array_like` for compatibility with Numpy docstrings.